src/result.hpp

```
@@ -64,6 +66,7 @@ struct InvariantMapPair {
 struct RelevantState {
```

Clang (14.0.6)

[warning] constructor does not initialize these fields: registers, stack_offsets

(cppcoreguidelines-pro-type-member-init)